app/scripts/modules/core/pipeline/config/stages/bake/bakeStageChooseOs.component.ts

```diff
@@ -0,0 +1,41 @@
+import {module} from 'angular';
+
+export class BakeStageChooseOSController implements ng.IComponentController {
+
+  public model: any;
+
+  static get $inject() {
+    return ['$scope'];
+  }
+
+  constructor(private $scope: ng.IScope) {}
+
+  public $onInit(): void {
+    this.$scope.model = this.model
+  }
+
+  public updateOnSelect(item: any): void {
+    this.model = item.id
+  }
+
+  public getBaseOsDescription(baseOsOption: any): String {
```

Comment:
Let's go ahead and declare an interface for `baseOsOption`, e.g.
```typescript
export interface IBaseOsOption {
  id: string;
  shortDescription?: string;
  detailedDescription: string;
  isImageFamily?: boolean;
}
```
This can go above the class in this same file - we're not very consistent about where we define these things.